plugins/org.jkiss.dbeaver.ext.oracle.ui/src/org/jkiss/dbeaver/ext/oracle/ui/editors/SchedulerJobActionEditor.java

```diff
@@ -45,7 +45,7 @@ protected String getSourceText(DBRProgressMonitor monitor) throws DBException {
     protected void setSourceText(DBRProgressMonitor monitor, String sourceText) {
         getInputPropertySource().setPropertyValue(
             monitor,
-            OracleConstants.PROP_OBJECT_BODY_DEFINITION,
+            OracleConstants.JOB_ACTION_DEFINITION,
```

Comment:
I don't understand how changing a constant fixes anything. Can you elaborate?

Reply:
During debugging, getInputPropertySource() doesn’t return any properties such as PROP_OBJECT_BODY_DEFINITION.